pact/concrete-policies/royalty-policy/royalty-policy-v1.pact

```diff
@@ -46,9 +46,31 @@
     true
   )
 
+  (defcap UPDATE-ROYALTY (token-id:string creator:string creator-guard:guard)
+    @doc "Update royalty information for token"
+    @event
+    (with-read royalties token-id {'creator-guard:=cg}
+      (enforce-guard cg)
+    )
+  )
+
   (defun get-royalty:object{royalty-schema} (token:object{token-info})
     (read royalties (at 'id token))
   )
+  
+  (defun rotate:string (token-id:string creator:string creator-guard:guard)
+    @doc "Rotates the royalty creator account and guard for the token"
+    (with-capability (UPDATE-ROYALTY token-id creator creator-guard)
+      (update royalties token-id {'creator:creator,
+                                  'creator-guard:creator-guard})
+    )
+    (emit-event (UPDATE-ROYALTY token-id creator creator-guard))
```

Comment:
Because the cap's written as an `@event` cap, it will be emitted automatically and this line is not necessary.